install/install.py

```diff
@@ -114,16 +115,17 @@ def install(self, cmd_arguments):
                 break
 
             print("")
-            for i in range(0, len(cp.sections())):
-                print("(%d) %s"%(i+1, cp.sections()[i]))
-            print("(0) %s (%s)"%("My distro is not listed :(", distro))
-            distro_num = raw_input("Select a number based on your distribution : ")
+            for i, item in enumerate(cp.sections()):
+                Colorizer.warning("(%d) %s" % (i + 1, item))
+            Colorizer.warning("(0) %s (%s)" % ("My distro is not listed :(", distro))
```

Comment:
Why formatting the first parameter? Since it is a hardcoded string, it should be part of the string directly, like: `'(0) My distro is not listed :( (%s)' % distro`